### PR TITLE
Helm: Add config for svc-bie-kafka

### PR DESCRIPTION
**Pull request by yoomlam**
## What was the problem?

Need Helm configuration for svc-bie-kafka in order to deploy to LHDI

Associated tickets or Slack threads:
- #1791

## How does this fix it?

Add Helm config for svc-bie-kafka

## How to test this PR
- Run [Update Deployment](https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/workflows/update-deployment.yml) for svc-bie-kafka in LHDI dev.
- Check that it is running in LHDI

### CODE REVIEW

File: app/docker-compose.yml [hunk 1]
```diff
@@ -113,7 +113,7 @@ services:
     image: va/abd_vro-svc-bie-kafka:latest
     <<: [ *common-sde-security, *common-security-opt ]
     environment:
-      <<: [ *rabbitmq-placeholder-vars ]
+      <<: [*rabbitmq-placeholder-vars, *common-vars]
```
Comment (yoomlam) [inline app/docker-compose.yml:116]: `common-vars` sets `ENV` which enables `application-$ENV.yaml` to be loaded.

File: helm/svc-bie-kafka/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          env:
+            {{- include "vro.commonEnvVars" . | nindent 12 }}
+            {{- include "vro.mqClient.envVars" . | nindent 12 }}
+            # VRO_SECRETS_BIE_KAFKA provides Kafka credentials (keys and certs)
+            # TODO: Uncomment the following after it is added to Hashicorp Vault and secrets are deployed
+            #- name: VRO_SECRETS_BIE_KAFKA
+            #  valueFrom:
+            #    secretKeyRef:
+            #      name: vro-secrets
+            #      key: VRO_SECRETS_BIE_KAFKA
```
Comment (yoomlam) [inline helm/svc-bie-kafka/templates/deployment.yaml:33]: Remember to uncomment after it is added to Hashicorp Vault and secrets are deployed. See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Secrets-Vault#addingmodifying-a-secret

File: helm/vro-app/templates/deployment.yaml [hunk 3]
```diff
@@ -89,6 +89,7 @@ spec:
                   name: vro-secrets
                   key: VRO_SECRETS_SLACK
 
+            # These 2 should be moved to application-*.yml
```
Comment (yoomlam) [inline helm/vro-app/templates/deployment.yaml:92]: Reminder that these env variables don't need to be in a Helm config. See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Configuration-settings

File: svc-bie-kafka/src/main/resources/application.yaml [hunk 4]
```diff
@@ -1,4 +1,8 @@
 spring:
+  profiles:
+    # See https://github.com/department-of-veterans-affairs/abd-vro/wiki/Configuration-settings#vros-use-of-spring-profiles
+    active: ${ENV:default}
```
Comment (yoomlam) [inline svc-bie-kafka/src/main/resources/application.yaml:4]: Causes `application-$ENV.yaml` to be loaded. In LHDI, `ENV` is set to `dev`, `qa`, `prod-test`, ... -- see https://github.com/department-of-veterans-affairs/abd-vro/wiki/Development-environments

File: svc-bie-kafka/src/main/resources/application-dev.yaml
Comment (yoomlam) [inline svc-bie-kafka/src/main/resources/application-dev.yaml:1]: Referring to https://github.com/department-of-veterans-affairs/abd-vro/issues/1674#issuecomment-1618814464, there doesn't seem to be corresponding BIE envs for LHDI's `qa` and `sandbox` envs, so I didn't create `application-$ENV.yaml` for those.

**Review by msnwatson — APPROVED** [review]

**Review by dfitchett — APPROVED** [review]

**Pull request merged**